packages/material-ui/src/RadioGroup/RadioGroup.test.js

```diff
@@ -1,170 +1,184 @@
 import React from 'react';
 import { assert } from 'chai';
-import { spy } from 'sinon';
-import { createShallow, createMount } from '@material-ui/core/test-utils';
+import { spy, stub } from 'sinon';
+import { createMount, findOutermostIntrinsic, testRef } from '@material-ui/core/test-utils';
 import FormGroup from '../FormGroup';
 import Radio from '../Radio';
 import RadioGroup from './RadioGroup';
 import consoleErrorMock from 'test/utils/consoleErrorMock';
 
 describe('<RadioGroup />', () => {
-  let shallow;
   let mount;
 
   before(() => {
     mount = createMount();
-    shallow = createShallow();
   });
 
   after(() => {
     mount.cleanUp();
   });
 
+  function findRadio(wrapper, value) {
+    return wrapper.find(`SwitchBase[value="${value}"]`).first();
+  }
+
+  function findRadioInput(wrapper, value) {
+    return wrapper
+      .find(`SwitchBase[value="${value}"]`)
+      .first()
+      .find('input');
+  }
+
+  it('does forward refs', () => {
+    testRef(<RadioGroup />, mount);
```

Comment:
@eps1lon Should the third argument be optional in the TypeScript definition as it has a default value? Webstorm complains that there should be 3 arguments.

Reply:
That's optional.

Reply:
Sorry, I wasn't really clear. I was suggesting this
```diff
export default function testRef<T>(
  element: React.ReactElement<{ innerRef: React.RefObject<T> }>,
  mount: ReturnType<typeof createMount>,
-  onRef: (ref: T) => void,
+  onRef?: (ref: T) => void,
): void;
```

Reply:
Sure. I didn't know webstorm actually type checks the js files. Shouldn't that throw up all over our codebase?

Reply:
Erm... It probably does 😁 I’ll look out for them.